Source/NSStoryboard.m

```diff
@@ -0,0 +1,975 @@
+
+- (NSString *) selector;
+- (void) setSelector: (NSString *)s;
+
+- (SEL) action;
+- (void) setAction: (SEL)action;
+
+- (id) sender;
+- (void) setSender: (id)sender;
+
+- (NSString *) identifier;
+- (void) setIdentifier: (NSString *)identifier;
+
+- (NSString *) kind;
+- (void) setKind: (NSString *)kind;
+@end
+
+@implementation NSStoryboardSeguePerformAction
+- (id) target
+{
+  return _target;
+}
+
+- (void) setTarget: (id)target
+{
+  ASSIGN(_target, target);
+}
+
+- (SEL) action
+{
+  return _action;
+}
+
+- (void) setAction: (SEL)action
+{
+  _action = action;
+}
+
+- (NSString *) selector
+{
+  return NSStringFromSelector(_action);
+}
+
+- (void) setSelector: (NSString *)s
+{
+  _action = NSSelectorFromString(s);
+}
+
+- (id) sender
+{
+  return _sender;
+}
+
+- (void) setSender: (id)sender
+{
+  ASSIGN(_sender, sender);
+}
+
+- (NSString *) identifier
+{
+  return _identifier;
+}
+
+- (void) setIdentifier: (NSString *)identifier
+{
+  ASSIGN(_identifier, identifier);
+}
+
+- (NSString *) kind
+{
+  return _kind;
+}
+
+- (void) setKind: (NSString *)kind
+{
+  ASSIGN(_kind, kind);
+}
+
+- (id) nibInstantiate
+{
+  return self;
+}
+
+- (IBAction) doAction: (id)sender
+{
+  [_sender performSegueWithIdentifier: _identifier
+                               sender: _sender];
+}
+
+- (id) copyWithZone: (NSZone *)z
+{
+  NSStoryboardSeguePerformAction *pa = [[NSStoryboardSeguePerformAction allocWithZone: z] init];
+  [pa setTarget: _target];
+  [pa setSelector: [self selector]];
+  [pa setSender: _sender];
+  [pa setIdentifier: _identifier];
+  return pa;
+}
+
+- (instancetype) initWithCoder: (NSCoder *)coder
+{
+  self = [super init];
+  if ([coder allowsKeyedCoding])
+    {
+      if ([coder containsValueForKey: @"NSTarget"])
+        {
+          [self setTarget: [coder decodeObjectForKey: @"NSTarget"]];
+        }
+      if ([coder containsValueForKey: @"NSSelector"])
+        {
+          [self setSelector: [coder decodeObjectForKey: @"NSSelector"]];
+        }
+      if ([coder containsValueForKey: @"NSSender"])
+        {
+          [self setSender: [coder decodeObjectForKey: @"NSSender"]];
+        }
+      if ([coder containsValueForKey: @"NSIdentifier"])
+        {
+          [self setIdentifier: [coder decodeObjectForKey: @"NSIdentifier"]];
+        }
+      if ([coder containsValueForKey: @"NSKind"])
+        {
+          [self setKind: [coder decodeObjectForKey: @"NSKind"]];
+        }
+    }
+  return self;
+}
+
+- (void) encodeWithCoder: (NSCoder *)coder
+{
+  // this is never encoded directly...
+}
+@end
+
+
+@interface NSControllerPlaceholder : NSObject <NSCoding, NSCopying> // , NSSeguePerforming>
+{
+  NSString *_storyboardName;
+  NSStoryboard *_storyboard;
+}
+
+- (NSString *) storyboardName;
+- (void) setStoryboardName: (NSString *)name;
+- (id) instantiate;
+
+@end
+
+@implementation NSControllerPlaceholder
+
+- (instancetype) init
+{
+  self = [super init];
+  if (self != nil)
+    {
+      _storyboardName = nil;
```

Comment:
This is the default in Objective C, no code is needed for that.

Reply:
Removed